src/views/Change.vue

```diff
@@ -149,6 +153,11 @@ export default class Change extends Mixins(
     return null
   }
 
+  /** Is true when the certify conditions are not met. */
+  get isCertifyInvalid (): boolean {
+    return this.getValidateSteps && !(this.getCertifyState.certifiedBy && this.getCertifyState.valid)
+  }
```

Comment:
I think you don't need this, since it's already computed in the sub-component, and also THIS component doesn't need to know this state.